design/src/App.scss

```diff
@@ -120,6 +120,137 @@ $ribbon-color: #9a0017;
     }
 }
 
+//Form
+.img-overlay {
+  width: 100vw;
+  max-width: 100%;
+  position: absolute;
+  height: 100vh;
+  background-color: rgba(0,0,0,0.7);
+  z-index: 100;
+}
+
+.form-section{
+  width: 100vw;
+  max-width: 100%;
+  position: relative;
+  height: 100vh;
+  background-image: url('../../img/background.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form-wrapper{
+  width: 25%;
+  height: auto;
+  background-color: rgba(157,132,113,0.9);
+  border-radius: 30px 0px 30px 0px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  padding:20px 0px;
+  position: relative;
+  z-index: 101;
+}
+.form-wrapper h3{
+  font-family: 'Merienda';
+  font-weight: bold;
+  color:white;
+  font-size: 1.3em;
+  text-shadow: 1px 1px #333;
+}
+
+.form-wrapper input[type="text"]{
+  width: 100%;
+  border-radius: 15px;
+  color:black;
+  padding: 15px 20px;
+  border:none;
+  margin:18px 0px;
+  background-color:#d2d2d2;
+  font-family: 'Lato';
+  font-weight: bold;
+}
+.form-wrapper input::-webkit-input-placeholder { /* Edge */
+  color: black;
+}
+
+.form-wrapper input:-ms-input-placeholder { /* Internet Explorer 10-11 */
```

Comment:
IE11 was "killed" in August 2020, so although backward compatibility is great (👍 ), I think this can be safely removed, unless there's a specific reason to keep it.

Reply:
Will do :)